seaborn/categorical.py

```diff
@@ -1835,6 +1836,10 @@ def __init__(self, x, y, hue, data, order, hue_order,
 
         self.outlier_prop = outlier_prop
 
+        if showfliers is None:
```

Comment:
Is this needed? The function signature can have `showfliers=True` as the default parameter.